code/modules/antagonists/heretic/heretic_antag.dm

```diff
@@ -67,7 +67,9 @@
 		PATH_BLADE = "label", // my favorite color is label
 		PATH_COSMIC = "purple",
 		PATH_KNOCK = "yellow",
+		PATH_MOON = "blue"
```

Comment:
```suggestion
		PATH_MOON = "blue",
```

Reply:
Before I change this, do you have any recommendations for another color to use?

Reply:
Nope